src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareIncludeExcludeNodeOperation.cs

```diff
@@ -38,6 +38,8 @@ class SchemaCompareIncludeExcludeNodeOperation : ITaskOperation
 
         public bool Success { get; set; }
 
+        public List<DiffEntry> ChangedDifferences;
```

Comment:
Any particular reason for explicit accessors in the other changes, but not here?